diffkemp/output.py

```diff
@@ -15,10 +112,15 @@ def __init__(self, snapshot_dir_old, snapshot_dir_new, result):
         self.output = {}
         # Sets with symbol names
         self.function_names = set()
-        self.macro_names = set()
         # Note: type_names contains tuples (type name, parent name)
+        #   parent name - name of function in which type is used
```

Comment:
Relevant comment from #314:
> Could be a macro, right?
> ```suggestion
>         #   parent name - name of function/macro in which type is used
> ```

I think, that currently, we are reporting type differences only from functions. Even if the type was originally used in a macro, the call stacks report type differences from the function in which the macro with the differing type was used. So the parent name (meaning the name of the previous call from the call stack) for type/struct should always be a function.